sycl/test/basic_tests/access_to_subset.cpp

```diff
@@ -1,8 +1,9 @@
-// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
Can you please clarify if `-fsycl-targets` is a mandatory argument for user to provide now? If yes, wouldn't it be better if it had a default value?

Reply:
Is not, it is only required on the test so we can compile correctly for CUDA or SPIRV targets.